source/common/formatter/substitution_formatter.cc

```diff
@@ -980,8 +990,9 @@ GrpcStatusFormatter::formatValue(const Http::RequestHeaderMap&,
 
 MetadataFormatter::MetadataFormatter(const std::string& filter_namespace,
                                      const std::vector<std::string>& path,
-                                     absl::optional<size_t> max_length)
-    : filter_namespace_(filter_namespace), path_(path), max_length_(max_length) {}
+                                     absl::optional<size_t> max_length, const bool unquoted)
```

Comment:
nit: I don't think we usually use const parameters that aren't references or pointers.

Reply:
not in headers, but it's used on definitions here and there:

```
 $ git grep 'const bool ' -- '*.cc' | grep :: | grep -v test/ | grep -v ';'
source/common/formatter/substitution_formatter.cc:                                     absl::optional<size_t> max_length, const bool unquoted)
source/extensions/filters/common/ext_authz/check_request_utils.cc:                                           const std::string& service, const bool local,
source/extensions/filters/network/redis_proxy/command_splitter_impl.cc:void SplitRequestBase::updateStats(const bool success) {
source/extensions/filters/network/thrift_proxy/router/router_impl.cc:void Router::UpstreamRequest::releaseConnection(const bool close) {
source/extensions/filters/network/zookeeper_proxy/filter.cc:void ZooKeeperFilter::onConnect(const bool readonly) {
source/extensions/filters/network/zookeeper_proxy/filter.cc:void ZooKeeperFilter::onGetDataRequest(const std::string& path, const bool watch) {
source/extensions/filters/network/zookeeper_proxy/filter.cc:void ZooKeeperFilter::onGetChildrenRequest(const std::string& path, const bool watch,
source/extensions/filters/network/zookeeper_proxy/filter.cc:void ZooKeeperFilter::onExistsRequest(const std::string& path, const bool watch) {
source/extensions/filters/udp/udp_proxy/udp_proxy_filter.cc:    const bool ok = Network::Socket::applyOptions(
source/extensions/quic_listeners/quiche/active_quic_listener.cc:    const bool ok = Network::Socket::applyOptions(
source/extensions/stat_sinks/common/statsd/statsd.cc:                             Network::Address::InstanceConstSharedPtr address, const bool use_tag,
source/extensions/tracers/zipkin/span_buffer.cc:JsonV2Serializer::JsonV2Serializer(const bool shared_span_context)
source/extensions/tracers/zipkin/span_buffer.cc:ProtobufSerializer::ProtobufSerializer(const bool shared_span_context)
source/server/admin/prometheus_stats.cc:    Buffer::Instance& response, const bool used_only, const absl::optional<std::regex>& regex,
source/server/admin/prometheus_stats.cc:    const bool used_only, const absl::optional<std::regex>& regex) {
source/server/admin/stats_handler.cc:                          const bool used_only, const absl::optional<std::regex> regex,
```

but yeah not super popular.